modules/aws-bastion-instance/example/variables.tf

```diff
@@ -1,3 +1,7 @@
 variable "vpc_id" {
   type = string
 }
+
+variable "public_subnet_id" {
```

Comment:
Add `description`. Remove `vpc_id` and `public_subnet_tags`, as they're unnecessary now.

Reply:
??
`public_subnet_tags` already removed
`vpc_id` is necessary to create sg.

Reply:
Ah, missed that.

Just a comment: That's why I felt no need to specify `subnet_id` initially, since it is simpler to just pass proper `public_subnet_tags`. No need to lookup `subnet_id` yourself then.